addons/stock/static/src/js/inventory_singleton_list_controller.js

```diff
@@ -51,9 +51,9 @@ var SingletonListController = InventoryReportListController.extend({
         var similarRecords = findSimilarRecords(handle);
 
         if (similarRecords.length > 1) {
-            var notification = _t("You tried to create a record who already exists."+
-            "<br/>This last one has been modified instead.");
-            this.do_notify(_t("This record already exists."), notification);
+            var notification = _t("You tried to create a record who already exists. "+
+            "This last one has been modified instead.");
```

Comment:
"You have tried to create a record which already exists"

Reply:
Ok, I'll take the opportunity to change it.